server/models/Profile.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const profileSchema = new Schema({
+    first_name: { type: String, required: true },
+    last_name: { type: String, required: true },
+    description: { type: String, required: true },
+    location: { type: String, required: true },
+    availability: [{
+        from: Date,
+        to: Date
+    }],
+    gender: { type: String, required: true },
+    profile_picture: { type: String, required: false },
+    phone_number: [{ type: Number, required: true }],
+    payment_methods: [{
+        card_type: String,
+        card_number: String, //encrypt later with bcrypt, do not store without encryption
+        card_expiration_date: String, //encrypt later with bcrypt, do not store without encryption
+        card_security_pin: Number, //encrypt later with bcrypt, do not store without encryption
+    }],
+    messages: [{
+        type: Schema.Types.ObjectId,
+        ref: ('Message')
+    }],
```

Comment:
I would prefer not to store messages in the profile model as it's not really related to the profile, we can add just a reference of the profile onto the message model when we get to that but for now let's remove this.